pkg/image/api/helper.go

```diff
@@ -191,6 +178,61 @@ func ParseDockerImageReference(spec string) (DockerImageReference, error) {
 	return ref, nil
 }
 
+// ParseDockerImageReference parses a Docker pull spec string into a
+// DockerImageReference.
+func ParseDockerImageReference(spec string) (DockerImageReference, error) {
+	// TODO replace with docker version once docker/docker PR11109 is merged upstream
```

Comment:
If you think it's not worth it, just delete the comment. On the other hand, I'd like to have a way to easily turn our `DockerImageReference` into upstream's `reference.Reference`.